modulus/metrics/general/crps.py

```diff
@@ -124,6 +134,82 @@ def _crps_gaussian(mean: Tensor, std: Tensor, obs: Union[Tensor, np.ndarray]) ->
     return std * (2 * phi + d * Phi - 1.0 / torch.sqrt(torch.as_tensor(torch.pi)))
 
 
+@torch.jit.script
+def _crps_from_empirical_cdf(
```

Comment:
Looks the same as e2mip, but you added support for arbitrary dim. Make sense to me.